src/lib/ProductImporter.ts

```diff
@@ -242,14 +261,19 @@ export class ProductImporter {
         await this.upsertBulkObject(conn, 'enxCPQ__AttributeSetAttribute__c', this.attributeSetAttributes);
         await this.deleteBulkObject(conn, 'enxCPQ__ProductAttribute__c', this.productAttributesIds);
         await this.upsertBulkObject(conn, 'enxCPQ__ProductAttribute__c', this.productAttributes);
+        // dlaczego list prdAttributeValues i attributeValues nie połączyć i nie robić jednego upserta?
         await this.upsertBulkObject(conn, 'enxCPQ__AttributeValue__c', this.prdAttributeValues);
         await this.upsertBulkObject(conn, 'enxCPQ__AttributeValue__c', this.attributeValues);
+        // A) w metodzie all nie powinno się dziać zadne filtrowanie, czyste sterowanie przepływem dla czytelnosci
+        // B) a w zasadzie jest jakiś powód dla którego nie update'ujemy standard price booka?
```

Comment:
powodem dla którego nie updatujemy Standard pricebooka jest to, że nie ma tech_external_id. Możemy go updatować po jakimś innym polu tylko pytanie po jakim?